<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualSettings</class>
 <widget class="QDialog" name="VisualSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visual settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/appIcon.svg</normaloff>:/resources/appIcon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Box height:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="height">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Box width:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="width">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Font:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFontComboBox" name="font"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Font size:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="fontSize"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Rounding:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="rounding">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Line width:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="lineWidth">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Line color:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="lineColor">
         <property name="text">
          <string>Change color</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Text color:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="textColor">
         <property name="text">
          <string>Change color</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Proband color:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="probantColor">
         <property name="text">
          <string>Change color</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Promoted color:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="promotedColor">
         <property name="text">
          <string>Change color</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Standard color:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="standardColor">
         <property name="text">
          <string>Change color</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Highlighted color:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="highlightedColor">
         <property name="text">
          <string>Change color</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Test:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Text:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLineEdit" name="textEdit"/>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Number of lines:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QSpinBox" name="lineNumber"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>graphicsView</tabstop>
  <tabstop>height</tabstop>
  <tabstop>width</tabstop>
  <tabstop>font</tabstop>
  <tabstop>fontSize</tabstop>
  <tabstop>probantColor</tabstop>
  <tabstop>promotedColor</tabstop>
  <tabstop>standardColor</tabstop>
  <tabstop>highlightedColor</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>lineNumber</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VisualSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
